modules/govuk/manifests/node/s_cache.pp

```diff
@@ -58,7 +58,7 @@
   }
 
   # Set the varnish storage size to 75% of memory - 1024
```

Comment:
(Personally I think the comment is pretty useless as the code is entirely clear, so maybe removing it is best)